aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -975,6 +975,12 @@ std::tuple<Tensor&, Tensor&> frexp_out(const Tensor& self,
   return std::tuple<Tensor&, Tensor&>(mantissa, exponent);
 }
 
+bool _is_true(const Tensor& self) {
+  TORCH_INTERNAL_ASSERT(self.numel() == 1);
```

Comment:
We don't necessarily have to have this requirement--we could do `self.all().item<bool>()` below instead. Not sure which way is better

Reply:
`self.all().item<bool>()` makes it a bit easier to write the batching rule. Though, I would expect an op named `_is_true` to just check if the value is True or False and not say something about all the values in the Tensor (that would be `_is_all_true`).

The batching rule for `_is_true(x)` is probably `_is_true(unwrap(x).all())`.

If we change the semantics to `_is_all_true(x)`, then its batching rule is literally just itself, `_is_all_true(unwrap(x))`.

Reply:
Ok I think `_is_all_true` seems a little better, I'll update to that